.golangci.yml

```diff
@@ -0,0 +1,20 @@
+name: Run Gosec
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
+      - name: Run Gosec Security Scanner
+        uses: securego/gosec@master
+        with:
+          args: ./...
```

Comment:
.golangci.yml is the [configuration file of golangci-lint](https://golangci-lint.run/usage/configuration/#config-file). You here pasted a GitHub workflow.

Reply:
Oh my bad. I pushed a new commit to enable `gosec`.